bridge/github/config.go

```diff
@@ -220,32 +262,59 @@ func randomFingerprint() string {
 	return string(b)
 }
 
-func promptTokenOptions(owner, project string) (string, error) {
+func promptTokenOptions(repo repository.RepoCommon, owner, project string) (*core.Token, error) {
 	for {
+		tokens, err := core.LoadTokens(repo)
+		if err != nil {
+			return nil, err
+		}
+
 		fmt.Println()
 		fmt.Println("[1]: user provided token")
 		fmt.Println("[2]: interactive token creation")
+		fmt.Println("known tokens for Github:")
+
+		var githubTokens []*core.Token
+		i := 0
+		for _, token := range tokens {
```

Comment:
i think you can use the index given by the range

Reply:
also, it seems you need something like `LoadTokenWithTarget` in the core to simplify that

Reply:
Since the tokens list can contain tokens with different targets doesn't provide the correct index to show. `LoadTokenWithTarget` will definitely solve this :+1: